docs/source/model_design/risk_factors.rst

```diff
@@ -1,55 +1,299 @@
-
-  It looks like most of that content is generalizable to both risks and causes.
-  Should we put similar sections in both documents, or pull them out into a more
-  general document about designing Vivarium models?
-
-  On the other hand, it looks like the following sections in :ref:`Modeling
-  Causes <models_cause>` have details that are specific to causes, but should
-  have analogues for risks in this document:
-
-  * How is a cause model incorporated into a larger model?
-  * What does a model document look like?
-
-.. contents::
-  :local:
-
-Overview
---------
-
-What is a risk factor?
-++++++++++++++++++++++
-
-What is a causal relationship?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-What is a risk exposure?
-++++++++++++++++++++++++
-
-What is a risk effect?
-++++++++++++++++++++++
-
-Definitions
------------
-
-Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
-++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-
-Relative Risk (RR)
-++++++++++++++++++
-
-Population Attributable Fraction (PAF)
-++++++++++++++++++++++++++++++++++++++
+.. _models_risk_factors:
+
+=====================
+Modeling Risk Factors
+=====================
+
+.. todo::
+
+  Add description of what's in this document.
+
+  Also see the following sections in :ref:`Modeling Causes <models_cause>`:
+
+  * Learning objectives
+  * Why do we want a document that describes each cause model?
+
+  It looks like most of that content is generalizable to both risks and causes.
+  Should we put similar sections in both documents, or pull them out into a more
+  general document about designing Vivarium models?
+
+  On the other hand, it looks like the following sections in :ref:`Modeling
+  Causes <models_cause>` have details that are specific to causes, but should
+  have analogues for risks in this document:
+
+  * How is a cause model incorporated into a larger model?
+  * What does a model document look like?
+
+.. contents::
+  :local:
+
+Overview
+--------
+
+What is a risk factor?
+++++++++++++++++++++++
+
+What is a causal relationship?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What is a risk exposure?
+++++++++++++++++++++++++
+
+What is a risk effect?
+++++++++++++++++++++++
+
+Definitions
+-----------
+
+Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Measures of occurence
++++++++++++++++++++++
+
+This is a recap: Epidemiology is the study of the distribution and determinants of disease frequency in human populations. Simply put, it is the study of the *occurence* of illness. Measures of disease frequency are tools to describe how common an illness is (or outcome of an event) with reference to the size of the population at risk. They are used to count cases, in relation to a population and to a measure of time. Outcomes can be infection, disease, disability, death, other risk-exposures, recovery or usage of health care. 

```

Comment:
I feel like we have a lot of this information documented throughout these pages, and to my mind it would be most helpful if this section zeroed in a bit on measures for comparing risk in different populations

Reply:
ya, it would be good to discuss as a group what we can keep here and what can be thinned out.